superset-frontend/src/views/CRUD/welcome/ChartTable.tsx

```diff
@@ -143,10 +142,10 @@ function ChartTable({
         buttons={[
           {
             name: (
-              <IconContainer>
-                <Icon name="plus-small" />
+              <div>
+                <i className="fa fa-plus" />
```

Comment:
A little sad about moving away from the custom icon to the font-awesome one, but I see why you're trying to whittle out the weird div. 

@geido should we just not worry about this for now, and tackle it after your icon work goes through?

Reply:
Sure 👍

Reply:
Ok. Here is the problem. We have a lot of places where a button is defined with a nested `<i>` element referencing an icon. These icons don't have any margin or padding, it's the real icon size. That's why this code here in `Button` automatically creates a margin between the icon and the text:
```
'i:first-of-type, svg:first-of-type': {
   marginRight: theme.gridUnit * 2,
   padding: `0 ${theme.gridUnit * 2} 0 0`,
},
```
The `Icon` component have a predefined `viewBox` of `24x24`, so that margin is not required. I could replace all `<i>` elements with the `Icon` component and remove the margin but as you mentioned @geido is already working on that. So I think the safest path is to make this modification in Icon PR and also remove `Button` automatic margin there.